r2i/ncsdk/frameworkfactory.cc

```diff
@@ -50,7 +50,8 @@ std::unique_ptr<r2i::IFrame> FrameworkFactory::MakeFrame (
 r2i::FrameworkMeta FrameworkFactory::GetDescription (RuntimeError &error) {
   const FrameworkMeta meta {
     .code = r2i::FrameworkCode::NCSDK,
-    .name = "NCSDK",
+    .name = "Ncsdk",
```

Comment:
Wouldn't this be NCSDK? Considering that other frameworks with acronyms have them in upper case.